<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVideoPlayer</class>
 <widget class="QWidget" name="CVideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1201</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>740</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed of Progress</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>750</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>730</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>740</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>x1.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>x1.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>x2.0</string>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>660</y>
     <width>971</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
